### DISCUSSION TITLE
Pulling off another great month in Pull Requests 💥 [Pull Requests Mini Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#64874] [Pull Requests]
![Header for the Pull Requests category check-in shows a blue starry background with various planets in the distance. The left has some clouds and an octocat in a Hawaiian shirt holding a GitHub mug. To the right there is a blue and red rocket ship taking off.](https://github.com/community/community/assets/92824518/b7f50bb7-5979-4c6f-92aa-8f3dc5347b99)

The feedback has been flowing and we’re so appreciative. Don’t miss a new opportunity to give feedback this month - detailed below. 

### ⚡Pull Requests Praise: Thanks to these fine folks for keeping our Community awesome ✔️
* @mujeeb00: For setting the record straight and giving us documentation feedback on [merge queue restrictions](https://github.com/orgs/community/discussions/62703). 
* @purajit: For all the feedback, links, and deep diving they did in their [Workflow to add PR to merge queue is slightly odd for Github Apps](https://github.com/orgs/community/discussions/62599). Love their meticulous approach!
   
🏅 Want to nominate a community member for a feature above? Just tag me (@queenofcorgis) where you’ve seen great community behavior.  🏅 

### 📣Feedback Wanted: Our team is looking for your thoughts! 
* **Share your thoughts on Copilot4PRs**: Do you have thoughts on what parts of the pull request experience could be automated? Share them with @aspeckofdustin in his [discussion](https://github.com/orgs/community/discussions/64219) 🧑‍✈️. 

### 🆒 Cool stuff™️ happening around GitHub and the Community
* **From the NFL to Software Engineer at GitHub**: @sgoldfarb2 shares her journey from working in video editing to coding for GitHub in her [blogpost](https://github.com/orgs/community/discussions/63383) 🏈.  
* **A checklist and guide to get your repository collaboration-ready**: In the world of software development, collaboration can make the difference between a brittle last-minute release and a reliable, maintainable, pain-free project, @peckjon share settings, content, and behavior to make your collaboration a success in his [blogpost](https://github.blog/2023-08-04-a-checklist-and-guide-to-get-your-repository-collaboration-ready/).  

### 🪐Universe 2023: Resources to get you ready
* **Universe 2023 tickets are available now**: Tickets are available (with an Early-Bird discount!). [Get tickets and learn more about Universe](https://github.com/orgs/community/discussions/63834).
* **Build for today, transform for tomorrow at GitHub Universe 2023**: Read our [blogpost](https://github.blog/2023-08-15-build-for-today-transform-for-tomorrow-at-github-universe-2023/) to get even more excited about Universe 2023. 

What are you hoping to pull off GitHub-wise this August? We’re all ears in the comments 👂.

**Comment by madisonscott** [comment]
Hey team, thanks for all your hard work! I'd love to see the "Hide whitespace" preference in diff view settings persist across PRs, as the Unified/Split preference does. I pretty much turn "Hide whitespace" on for every diff I look at!
<img width="399" alt="image" src="https://github.com/community/community/assets/10393950/e468f628-ca36-4326-8d5f-4e44745b428d">

**Reply by queenofcorgis** [reply]
Hey @madisonscott - thanks for this feedback, to make sure the right eyes are on it would you mind opening a new discussion in this category please?

**Reply by madisonscott** [reply]
Sure!

**Comment by queenofcorgis** [comment]
Fresh off the presses: [September PRs check-in](https://github.com/orgs/community/discussions/68541) 📰
